priv/graphql/schemas/admin/vcard.gql

```diff
@@ -0,0 +1,15 @@
+"""
+Allow admin to set user's vcard
+"""
+type VcardAdminMutation @protected{
+    "Set a new vcard for a user"
+    setVcard(user: JID!, vcard: VcardInput!): Vcard
```

Comment:
I forgot that we need to restrict domain admins to modify only their domain users' vcard.
```suggestion
    setVcard(user: JID!, vcard: VcardInput!): Vcard
      @protected(type: DOMAIN, args: ["user"])
```